### ISSUE TITLE
layers_get_feature_media() does not work for "page" post type. Featured video is never rendered!

### ISSUE THREAD

**Issue by dameer** [Obox/layerswp#315]
There's an option to add video to Page (beside Post) via Layers metabox - as an addition to Featured image. However, featured video on front-end is never rendered for pages. I've been researching a bit and figured out that it happens due to the "lack of condition" in layers_get_feature_media() function.

If you open "layerswp/core/helpers/template.php" and scroll down to line 1278 - 1283, you'll find the following conditions:
```
if( NULL != $postid &&
		(
			( is_single() && isset( $use_video ) ) ||
			( ( !is_single() && !is_page_template( 'template-blog.php' ) ) && isset( $use_video ) && !isset( $use_image) )
		)
) {
```
Because of the fact that "is_single()" doesn't check for "page" post type, it should rather be replaced with "is_singular()" which does:
```
if( NULL != $postid &&
		(
			( is_singular() && isset( $use_video ) ) ||
			( ( !is_singular() && !is_page_template( 'template-blog.php' ) ) && isset( $use_video ) && !isset( $use_image) )
		)
) {
```

**Comment by vailjoy** [comment]
Hi there,
I am fairly certain this is by design (we want to allow setting featured media for shares and widgets, but not force it output on the page). The video can be embedded in the page content with the same url placed wherever you want it to appear. I am double-checking with the team to get their input for you.

**Comment by dameer** [comment]
I just thought that if featured video metabox is found on both page and post edit screen - should be working on both Page and single Post templates. And in case it's not supposed to be working on Page template, maybe better to hide it on page edit screen - it's kinda confusing.

**Comment by vailjoy** [comment]
Fixed in 1.2.13
